Multisig/Features/Security/SecurityCenter.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  SecurityCenter.swift
+//  Multisig
+//
+//  Created by Mouaz on 1/4/23.
+//  Copyright © 2023 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+class SecurityCenter {
+    static let shared = SecurityCenter()
+
+    private let sensitiveStore = SensitiveStore.shared
```

Comment:
it would be better for dependencies to be injected. Into constructor for example. This will make this class better testable. Instead of lazy initialisation singleton could be created in AppDelegate.